### DISCUSSION TITLE
Issues running fast_Herbie_array() on ECMWF datasets

### DISCUSSION THREAD

**Discussion by v7nguyen** [blaylockbk/Herbie#72] [General]
ValueError: t already exists as coordinate or variable name.
![image](https://user-images.githubusercontent.com/53784737/165810613-d9b6a439-057a-43d0-b413-ce418b281317.png)

When running the function, I think since there is a "t" column for temperature that is causing the "xr" package to throw an error. I believe this is the error, because when I run t2m, it works like a charm. 
![image](https://user-images.githubusercontent.com/53784737/165815791-fd046875-b404-4c05-96fb-1a35996b1a76.png)

**Comment by blaylockbk** [comment]
That's another good find @v7nguyen.

Ok, I just updated the concat_dim names to `i_run` and `i_fxx`, so hopefully those wont conflict with any other variable names.
